src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -22,10 +22,20 @@ class PressableWithSecondaryInteraction extends Component {
             this.props.forwardedRef(this.pressableRef);
         }
         this.pressableRef.addEventListener('contextmenu', this.executeSecondaryInteractionOnContextMenu);
+        this.pressableRef.addEventListener('touchstart', this.preventDefault);
     }
 
     componentWillUnmount() {
         this.pressableRef.removeEventListener('contextmenu', this.executeSecondaryInteractionOnContextMenu);
+        this.pressableRef.removeEventListener('touchstart', this.preventDefault);
+    }
+
+    /**
+     * @param {touchstart} e - TouchEvent.
+     * https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent
+     */
+    preventDefault = (e) => {
```

Comment:
I don't think we support this pattern in Class component yet.

Reply:
Good catch! I will create a follow-up to clean this up!